package.json

```diff
@@ -8,6 +8,7 @@
     "lint": "vue-cli-service lint"
   },
   "dependencies": {
+    "bootstrap": "^5.3.0",
```

Comment:
Das ist eine super Idee. 

das package.json ist der Ort an dem Abhängigkeiten (zum Beispiel Plugins) zu deinem Projekt installiert werden können. 

Installiere noch popper dazu, dies benötigst du, um das Javascript von Bootstrap nutzen zu können.

das machst du in dem du folgendes hier als neue Zeile bei "dependencies" hinzufügst:

```
"@popperjs/core": "^2.11.8",
```

Wenn du das gemacht hast, muss du es natürlich auch noch installieren. Das machst du in dem du in deiner bash folgendes Kommando ausführst: 

```
npm install
```

dieses Kommando musst du immer ausführen, wenn du im `packange.json` etwas änderst.